src/i18n/langResources/lang.de.ts

```diff
@@ -1,38 +1,38 @@
 import { ILangResource } from '../../models/i18n';
 
 export const langDe: ILangResource = {
-  newVersionMessage: 'Willkommen zur neuen Version von vscode-icons!',
-  seeReleaseNotes: 'Was wurde geändert? (Englisch)',
-  dontShowThis: 'Diese Nachricht beim nächsten Update nicht mehr anzeigen',
-  seeReadme: 'Etwas über die Erweiterung lernen (Englisch)',
-  welcomeMessageBegin: 'vscode-icons benutzt nun die offizielle API. Aktiviere die Icons unter ',
-  welcomeMessageEnd: 'und wähle VSCode Icons aus.',
+  newVersionMessage: 'Willkommen zur neuen Version von vscode-icons.',
+  seeReleaseNotes: 'Informationen über die letzten Änderungen (en englisch)',
+  dontShowThis: 'Zeige mir diese Nachricht nicht das nächste Mal',
+  seeReadme: 'Erfahren Sie mehr über diese Erweiterung (en englisch)',
+  welcomeMessageBegin: 'vscode-icons benutzt nun die offizielle API. Gehen Sie zu ',
+  welcomeMessageEnd: 'und wählen Sie VSCode Icons, um zu aktivieren.',
   activationPath: {
     darwin: 'Code > Einstellungen > Datei-Symboldesign ',
     linux: 'Datei > Einstellungen > Datei-Symboldesign ',
     win32: 'Datei > Einstellungen > Datei-Symboldesign ',
   },
-  aboutOfficialApi: 'Mehr über Datei und Ordner Icons herausfinden (Englisch)',
-  learnMore: 'Mehr herausfinden!',
+  aboutOfficialApi: 'Erfahren Sie mehr über Datei und Ordner Icons (en englisch)',
+  learnMore: 'Wollen Sie mehr erfahren?',
   urlReleaseNote: 'https://github.com/robertohuertasm/vscode-icons/blob/master/CHANGELOG.md',
   urlReadme: 'https://github.com/robertohuertasm/vscode-icons/blob/master/README.md',
   urlOfficialApi: 'http://code.visualstudio.com/docs/customization/themes#_select-an-icon-theme',
   reload: 'Neu starten',
   autoReload: 'Automatisch neu starten',
   disableDetect: 'Projekterkennung deaktivieren',
-  iconCustomizationMessage: 'Das Icon-Manifest wird nun neu generiert...',
-  iconRestoreMessage: 'Das Icon-Manifest wird wieder zurückgesetzt.',
-  ngPresetMessage: 'Angular Icons wird ',
+  iconCustomizationMessage: 'Das Icon-Manifest wird regeneriert werden. ',
```

Comment:
It's both in the future. With `werden` it's just more explicitly that you are in the future, but it's clear out of the context. The other can both be present and future.